.github/workflows/integration.yml

```diff
@@ -41,14 +41,8 @@ jobs:
       - name: Install Dependencies
         run: pnpm i
 
-      - name: Biome
-        run: bun lint:biome
-
-      - name: ESLint
-        run: bun lint:eslint
-
-      - name: Type Check
-        run: bun lint:types
+      - name: Lint
+        run: bun run lint
```

Comment:
I do tend to prefer separate steps here. The reason being, when CI fails, you know why instantly, and it's often faster read the thing that failed ("ah, my types are bad"), then go read the message and apply the fix in your development environment